mode/sql/sql.js

```diff
@@ -96,7 +95,15 @@ CodeMirror.defineMode("sql", function(config, parserConfig) {
     } else if (operatorChars.test(ch)) {
       // operators
       stream.eatWhile(operatorChars);
-      return null;
+      return "operator";
+    } else if (brackets.test(ch)) {
+      // brackets
+      stream.eatWhile(brackets);
+      return "brckt";
```

Comment:
Please don't create a new token type for this—as you apparently noticed, `"bracket"` is already used, so you should use that.